src/spdx3/bump_from_spdx2/creation_information.py

```diff
@@ -24,9 +24,10 @@
 from spdx3.model.spdx_document import SpdxDocument
 
 
-def bump_creation_information(spdx2_creation_info: Spdx2_CreationInfo, payload: Payload) -> SpdxDocument:
+def bump_creation_information(spdx2_creation_info: Spdx2_CreationInfo, payload: Payload,
+                              document_namespace: str) -> SpdxDocument:
```

Comment:
As `document_namespace` is a field in `spdx2_creation_info` I don't think that we need it as an input argument here.